src/index.rs

```diff
@@ -44,7 +53,7 @@ impl Index {
         Self { inner: index }
     }
 
-    pub fn create_field(&self, name: &str, weight: f64, tag_options: Option<&TagOptions>) -> Field {
+    pub fn create_field(&self, name: &str, weight: f64, tag_options: &TagOptions) -> Field {
```

Comment:
I was wondering whether it makes sense to pass the options by reference. Usually you create options and want them to be consumed (unless you have a good reason to want to reuse them). Performance wise it's a small struct so doesn't make sense to pass by reference if not necessary.

Reply:
:+1: